pkg/crypto/tinkcrypto/crypto.go

```diff
@@ -151,3 +154,35 @@ func (t *Crypto) Verify(sig, msg []byte, kh interface{}) error {
 
 	return err
 }
+
+// ComputeMAC computes message authentication code (MAC) for code data
+// using a matching MAC primitive in kh key handle
+func (t *Crypto) ComputeMAC(data []byte, kh interface{}) ([]byte, error) {
+	keyHandle, ok := kh.(*keyset.Handle)
+	if !ok {
+		return nil, errBadKeyHandleFormat
+	}
+
+	macPrimitive, err := mac.New(keyHandle)
+	if err != nil {
+		return nil, err
+	}
+
+	return macPrimitive.ComputeMAC(data)
+}
+
+// VerifyMAC determines if mac is a correct authentication code (MAC) for data
+// using a matching MAC primitive in kh key handle and returns nil if so, otherwise it returns an error.
+func (t *Crypto) VerifyMAC(macBytes, data []byte, kh interface{}) error {
+	keyHandle, ok := kh.(*keyset.Handle)
+	if !ok {
+		return errBadKeyHandleFormat
+	}
+
+	macPrimitive, err := mac.New(keyHandle)
+	if err != nil {
+		return err
+	}
```

Comment:
I don't know to create the error condition here. I attempted passing an inappropriate key handle (for example, `keyset.NewHandle(aead.AES256GCMKeyTemplate()`), but unfortunately that causes the Tink code to panic instead of returning an err.

The only other thing I can think of is to mac that mac.New function a parameter (either for the function or in the Crypto struct) so that I can pass in a mock for the unit test, but I think that would be a bit ugly and perhaps confusing... let me know what you think.